frontend/src/container/GuestUpload.jsx

```diff
@@ -36,6 +38,27 @@ class GuestUpload extends React.Component {
     }
   }
 
+  async componentDidMount () {
+    const { props } = this
+    const request = await getGuestUploadInfos(props.match.params.token)
+    const response = await handleFetchResult(request)
+
+    switch (response.apiResponse.status) {
+      case 200:
+        this.setState({
+          hasPassword: response.body.has_password
```

Comment:
by using the fetchWrapper, response.body.has_password shall become response.json.has_password (If I'm not mistaken)